doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -467,10 +454,19 @@ arguments used for configuration.  Here is a table describing these emulated dev
 
    * - ``virtio-net``
      - Virtio network type device, parameter should be appended with the format:
-       ``virtio-net,<device_name>,[vhost],[mac=<XX:XX:XX:XX:XX:XX>]``.
-       The ``mac`` address is optional, ``device_name`` is the name of the TAP (or MacVTap) device.
-       It must include the keyword ``tap``.
+       ``virtio-net,<device_type=name>,[vhost],[mac=<XX:XX:XX:XX:XX:XX>],[mac_seed=<seed_string>]``.
```

Comment:
I'm assuming the use of `mac` and `mac_seed` is mutually exclusive, isn't it?

If so, we should make this clear in the doc,